test/test_models_detection.py

```diff
@@ -180,3 +181,14 @@ def test_compute_target_db():
     assert gts.shape[0] == masks.shape[0] == thresh_gts.shape[0] == thresh_masks.shape[0] == batch_size
     assert gts.shape[1] == masks.shape[1] == thresh_gts.shape[1] == thresh_masks.shape[1] == 1024
     assert gts.shape[2] == masks.shape[2] == thresh_gts.shape[2] == thresh_masks.shape[2] == 1024
+
+    return gts, masks, thresh_gts, thresh_masks
+
+
+def test_compute_loss_db(test_compute_target_db):
+    model = detection.db_resnet50(pretrained=False)
+    dbinput = tf.random.uniform(shape=[3, 1024, 1024, 3], minval=0, maxval=1)
+    proba_map, thresh_map, binary_map = model(dbinput, training=True)
+    gt, mask, thresh_gt, thresh_mask = test_compute_target_db
+    loss = model.compute_loss(proba_map, binary_map, thresh_map, gt, mask, thresh_gt, thresh_mask)
+    assert isinstance(loss, tf.Tensor)
```

Comment:
Perhaps we should also check that this is a valid loss (i.e. that we can backprop it?)